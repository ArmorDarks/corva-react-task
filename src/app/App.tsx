import { hot } from 'react-hot-loader/root'
import React from 'react'

const App = () => (
  <h1>Hello, world!</h1>
)

export default hot(App)
