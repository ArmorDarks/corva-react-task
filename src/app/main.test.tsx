/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'
import App from './App'

describe('main', () => {
  it('should be test', () => {
    expect(shallow(<App />)).toMatchSnapshot()
  })
})
