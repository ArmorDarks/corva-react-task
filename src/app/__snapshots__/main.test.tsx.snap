// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main should be test 1`] = `
<h1>
  Hello, world!
</h1>
`;
